### DISCUSSION TITLE
Sharing Metastore SQL Database with varying schemas

### DISCUSSION THREAD

**Discussion by pinkfloydx33** [datalust/seq-tickets#1546] [Help]
The documentation for using MS/Azure Sql or PostgreSQL for the metastore both indicate that you cannot share a database between multiple instances. However there is a `--schema` option (and corresponding configuration key) allowing you to specify the database schema the store should use.

The schema argument is not mentioned in the SQL migration docs otherwise (nor initial setup) which makes me wonder if it's a newer feature? If so, is it now possible to share a database for multiple instances so long as they use different schemas? I'd prefer not to have to provision a bunch of Azure dbs for my non-prod instances if I can just use a single one. I've migrated one instance over (with a custom schema) and there's only a single table created which makes it feel even more wasteful.

**Comment by nblumhardt** [comment]
Hi @pinkfloydx33; I believe it should be possible to share a single database with varying schemas. It's not documented as it wasn't a scenario we originally gave priority to, but at this point it I think we'll maintain support for it 👍
